platform/darwin/src/MGLFeature.mm

```diff
@@ -5,14 +5,30 @@
 #import "MGLPolygon.h"
 #import "MGLValueEvaluator.h"
 
+#import "MGLShape_Private.h"
 #import "MGLMultiPoint_Private.h"
+#import "MGLPolyline+MGLAdditions.h"
+#import "MGLPolygon+MGLAdditions.h"
+#import "NSDictionary+MGLAdditions.h"
+
+#import "NSExpression+MGLAdditions.h"
 
 #import <mbgl/util/geometry.hpp>
+#import <mapbox/geometry/feature.hpp>
+
+@interface MGLFeatureUtility: NSObject
```

Comment:
Fixed in https://github.com/mapbox/mapbox-gl-native/pull/6524/commits/206af315fd00efbe3f4b400aa7bd6317db408ac8